packages/parser/src/CombatData.ts

```diff
@@ -32,68 +32,146 @@ const SPELL_ID_TO_CLASS_MAP = new Map<string, CombatUnitClass>(
   }),
 );
 
-/*
-   Create a new interface to hold shuffle specific data
+/**
+ * Fields that describe a 2v2 or 3v3 conflict between players
+ * with a conclusive end as observed by one of the players in the match
+ */
+export interface IArenaCombat {
+  id: string;
+  wowVersion: WowVersion;
 
-   We can detect when we need this at the first event (_START) using "Rated Solo Shuffle" and "Solo Shuffle"
-   strings in the bracket column
-*/
-export interface IShuffleRound {
-  dataType: 'ShuffleRound';
-  wowVersion: 'retail';
+  /**
+   * Marker to discriminate types of combats into shuffle rounds or arena matches
+   */
+  dataType: 'ShuffleRound' | 'ArenaMatch';
+
+  /**
+   * Information decoded from ARENA_MATCH_START
+   */
   startInfo: ArenaMatchStartInfo;
 
+  /**
+   * All units that had log events during the match, includes players and pets
+   */
   units: { [unitId: string]: ICombatUnit };
+  /**
+   * Parsed events
+   */
   events: (CombatAction | CombatAdvancedAction)[];
+
+  /**
+   * Raw log lines of the underlying segment
+   */
   rawLines: string[];
+  /**
+   * Count of lines the combat log parser could not turn into higher level objects
+   */
   linesNotParsedCount: number;
+  /**
+   * Start time of the round, based on ARENA_MATCH_START
+   */
+  startTime: number;
+  /**
+   * End time of the round, based on player death inspection
+   */
+  endTime: number;
+
+  /**
+   * Team of player who recorded the match
+   */
+  playerTeamId: string;
+
+  /**
+   * Result of the match from the perspective of the recorder's team
+   *   Based on unit deaths for shuffles
+   *   Based on ARENA_MATCH_END event for regular arena matches
+   */
+  result: CombatResult;
+
+  /**
+   * Id of team who won, inferred from player death
+   */
   winningTeamId: string;
-  roundEndInfo: {
-    endTime: number;
-    killedUnitId: string; // Combatant whose death caused the round to end
-  };
-  scoreboard: { [unitId: string]: number }; // scoreboard at round-end of # of wins each unit has
 
-  sequenceNumber: number; // 0-5, which round in the 6 round sequence this is
+  /**
+   * Flag based on combat log data observed, an option in WoW's advanced settings
+   * Required to have position data in the logs
+   */
+  hasAdvancedLogging: boolean;
+
+  /**
+   * Information decoded from ARENA_MATCH_END
+   * This data will be unavailable for solo shuffle rounds that are recorded locally
+   * However - once the match ends the DTO will contain the endInfo object copied from
+   * the end of the match
+   *
+   * __Will not be available on locally recorded shuffles__
+   */
+  matchEndInfo?: ArenaMatchEndInfo;
+
+  /**
+   * Results of the match according to ARENA_MATCH_END
+   *
+   * __Will not be available on locally recorded shuffles__
+   */
+  matchResult?: CombatResult;
```

Comment:
there is a "result" field above. Do we still need this? If this is included in matchEndInfo, we can probably skip this field?